recipe_scrapers/schoolofwok.py

```diff
@@ -0,0 +1,172 @@
+from collections import defaultdict
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import IngredientGroup
+from ._utils import normalize_string
+
+
+class SchoolOfWok(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "schoolofwok.co.uk"
+
+    def author(self):
+        return "School of Wok"
+
+    def site_name(self):
+        return "School of Wok"
+
+    def title(self):
+        return self.soup.find("h1").get_text()
+
+    def cuisine(self):
+        categoryheader = self.soup.find(
+            string=lambda text: text and text.lower() == "cuisine"
+        )
+
+        if not categoryheader:
+            return ""
+
+        return categoryheader.find_next("p").get_text()
+
+    def total_time(self):
+        timeheader = self.soup.find(string=lambda text: text and text.lower() == "time")
+
+        if not timeheader:
+            return ""
+
+        return int(timeheader.find_next("p").get_text().split(" ")[0])
+
+    def yields(self):
+        servingheader = self.soup.find(
+            string=lambda text: text and text.lower() == "servings"
+        )
+
+        if not servingheader:
+            return ""
+
+        servingsize = servingheader.find_next("p").get_text().split(" ")[0]
+        return (
+            f"{servingsize} serving"
+            if int(servingsize) == 1
+            else f"{servingsize} servings"
```

Comment:
```suggestion
        return (
            get_yields(servingheader.find_next("p").get_text())
            if servingheader
            else None
```
Similarly this will now return None instead of "", also implemented the built in util `get_yields` which will help format the output and handles the same functionality as your custom implementation here by appending serving(s) correctly.